config/database.yml

```diff
@@ -5,21 +5,23 @@
 #   gem 'sqlite3'
 #
 default: &default
-  adapter: sqlite3
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
+  adapter: postgresql
+  encoding: unicode
+  pool: 5
+  username: postgres
+  password: password
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: <%= ENV['DB_DEVELOPMENT'] %>
 
 # Warning: The database defined as "test" will be erased and
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: db/test.sqlite3
+  database: <%= ENV['DB_TEST'] %>
```

Comment:
It makes sense now, :grin:  you can forget about my previous question from the top of the PR.